src/services/functionAppService.test.ts

```diff
@@ -29,7 +28,7 @@ describe("Function App Service", () => {
   const authKeyUrl = `${baseUrl}${app.id}/functions/admin/token?api-version=2016-08-01`;
   const syncTriggersUrl = `${baseUrl}${app.id}/syncfunctiontriggers?api-version=2016-08-01`;
   const listFunctionsUrl = `${baseUrl}${app.id}/functions?api-version=2016-08-01`;
-  const uploadUrl = `https://${app.enabledHostNames[0]}${constants.scmZipDeployApiPath}/`
+  const uploadUrl = `https://${app.enabledHostNames[1]}/api/zipdeploy/`;
```

Comment:
Why the switch from `enabledHostNames[0]` to `enabledHostNames[1]`? Might be good to document, even if it's just in the test

Reply:
I hardened the retrieval of the SCM domain from the list of domains in the website.  Based on that update I also changed the mock data used in the test and know that the SCM domain is the 2nd one in this case.

Reply:
Updated test to do similar logic.